packages/app/src/runs/modals/CreateCloudOrgModal.vue

```diff
@@ -101,29 +94,20 @@ const props = defineProps<{
 }>()
 
 const query = useQuery({
-  query: CheckCloudOrganizationsDocument,
+  query: CloudOrganizationsCheckDocument,
   requestPolicy: 'network-only',
+  pause: true,
 })
 
-const polling = ref(false)
+const refetch = useDebounceFn(() => query.executeQuery(), 1000)
 
-const { pause, resume } = useIntervalFn(() => {
-  if (props?.gql?.organizationControl?.nodes?.length || 0 > 0) {
-    pause()
-  } else {
-    query.executeQuery()
-  }
-}, 5000)
+const waitingOrgToBeCreated = ref(false)
 
-function startPolling () {
-  if (!polling.value) {
-    resume()
-    polling.value = true
-  }
+function startWaitingOrgToBeCreated () {
+  waitingOrgToBeCreated.value = true
 
   setTimeout(() => {
```

Comment:
Do we need to clean this `setTimeout` up (in `onUnmount`, or using the `useTimeout` function maybe?)